### ISSUE TITLE
Name "main::INC" used only once: possible typo

### ISSUE THREAD

**Issue by ntyni** [renormalist/data-dpath#17]
Since 0.57, just using Data::DPath will warn here:

```
% perl -Ilib -w -e 'use Data::DPath'
Name "main::INC" used only once: possible typo at /usr/share/perl/5.26/Safe.pm line 372.
```

This can be reduced to just

```
% perl -w -MSafe -e 'BEGIN { $c = Safe->new; $c->reval("no warnings") }'
Name "main::INC" used only once: possible typo at /usr/share/perl/5.26/Safe.pm line 372.
```

so some more Safe polishing seems to be in order.

Seen on Debian unstable, Perl 5.26.1. Original report is at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=877093

Thanks for your work on Data-DPath.

**Comment by renormalist** [comment]
@ntyni ACK.

**Comment by gregoa** [comment]
This has been fixed somewhere between 0.57 and 0.60.

**Comment by renormalist** [comment]
That's good news. Thanks for checking. So I can close this ticket.
